src/lib/THERMAL/rpm.cpp

```diff
@@ -0,0 +1,58 @@
+#include "targets.h"
+
+#if defined(PLATFORM_ESP32)
+#include "logging.h"
+#include <driver/pcnt.h>
+#include <soc/pcnt_struct.h>
+
+static volatile int overflow = 0;
+static uint32_t lastTime = 0;
+
+static void IRAM_ATTR pcnt_overflow(void *arg)
+{
+    overflow++;
```

Comment:
What happens when `overflow` overflows?
Doesn't it need to reset `overflow=0` somewhere?

Reply:
Yup